django_stomp/settings.py

```diff
@@ -19,3 +23,17 @@ def eval_settings_otherwise_raise_exception(
 STOMP_PROCESS_MSG_WORKERS = eval_settings_otherwise_raise_exception(
     "STOMP_PROCESS_MSG_WORKERS", eval_as_int_if_provided_value_is_not_none_otherwise_none
 )
+
+STOMP_SERVER_HOST = getattr(django_settings, "STOMP_SERVER_HOST", "127.0.0.1")
+STOMP_SERVER_PORT = eval_as_int_if_provided_value_is_not_none_otherwise_none(
+    getattr(django_settings, "STOMP_SERVER_PORT", 61613)
+)
+STOMP_USE_SSL = eval_str_as_boolean(os.getenv("STOMP_USE_SSL"))
```

Comment:
**suggestion** You can set the default value to `False`:

https://github.com/juntossomosmais/django-stomp/blob/master/django_stomp/services/consumer.py#L161

Reply:
Nice! I'll add!

Reply:
Added!